pkg/proc/proc_test.go

```diff
@@ -726,6 +726,7 @@ func TestNextFunctionReturnDefer(t *testing.T) {
 }
 
 func TestNextNetHTTP(t *testing.T) {
+	skipOn(t, "broken", "ppc64le")
```

Comment:
This test shouldn't be arch specific.

Reply:
100% right. I don't know why I skipped it, I just rerun it and it works perfectly fine.